main/forecast.py

```diff
@@ -5,7 +5,11 @@
 
 
 TIMESTAMP = datetime.datetime.today().strftime("%d.%m.%Y %H:%M:%S")
-API1 = 'https://www.theweathernetwork.com/us/api/data/usny0996/ci?ts=1752'
+CITY = ['NYC', 'SAN FRANCISCO', 'CHICAGO', 'MIAMI']
+API = ['https://www.theweathernetwork.com/us/api/data/usny0996/ci?ts=1752',
+       'https://www.theweathernetwork.com/us/api/data/usca0987/ci?ts=2152',
+       'https://www.theweathernetwork.com/us/api/data/usil0225/ci?ts=2152',
+       'https://www.theweathernetwork.com/us/api/data/usfl0316/ci?ts=2152']
 
```

Comment:
you use two separate data structures for the same user input. you should consider using dictionary instead